### DISCUSSION TITLE
Multiple config files?

### DISCUSSION THREAD

**Discussion by perllaghu** [coder/code-server#6592] [Q&A]
I'm adding code-server to a dockerised jupyter notebook service, and everything is building fine.

My current head-scratch is setting up config files: `~/.local/share/code-server` is in the users home directory, however that is an externally mounted volume and thus cannot be pre-defined in the docker image.

Is there a set of paths that the server looks in for config? [such as `/usr/local/etc/code-server/`, `/usr/local/share/code-server/`,  `~/.local/share/code-server/`, `~/.config/code-server/`]

**Comment by code-asher** [comment]
No, it only looks at one path for user data.  The default is `~/.local/share/code-server` as you have noted although it can be changed with `--user-data-dir`.

**Reply by code-asher** [reply]
There is also a config file (`~/.config/code-server/config.yaml`, can change with `--config`) but this is a separate thing, it has no overlap with the data directory.
